### DISCUSSION TITLE
Does not work in preview.

### DISCUSSION THREAD

**Discussion by tosdesign** [phcode-dev/phoenix#1984] [Q&A]
The preview will not work with the following responsive writing style.
Can I make it work in the settings?

.flex01{
	display: flex;
	align-items: flex-start;
	@media screen and (max-width: 768px) {
		flex-direction:column;
	}
}

**Comment by abose** [comment]
Can you give the exact steps to reproduce the issue. BTW, the media query will only work once the preview is resized to the given media width.

**Comment by tosdesign** [comment]
It is a matter of writing.
It is a writing style that works in normal browsers such as chrome,
but does not work in phoenix preview.

--------Not working--------

.flex01{
display: flex;
align-items: flex-start;
@media screen and (max-width: 768px) {
flex-direction:column;
}
}

--------It's okay--------

.flex01{
display: flex;
align-items: flex-start;
}

@media screen and (max-width: 768px) {
.flex01{
flex-direction:column;
}
}
